src/client/components/filter-menu/time-filter-menu/fixed-time-tab.tsx

```diff
@@ -40,46 +50,83 @@ export interface FixedTimeTabProps {
 }
 
 export interface FixedTimeTabState {
-  start: Date;
-  end: Date;
+  start?: Date;
+  end?: Date;
   shift: string;
+  activeMonthStartDate?: Date;
+  hoverTimeRange?: TimeRange;
+  selectionSet?: boolean;
+  dateRangePickers: List<any>;
+  timeRanges: List<any>;
+  numTimeRanges: int;
 }
 
 export class FixedTimeTab extends React.Component<FixedTimeTabProps, FixedTimeTabState> {
-
   initialState = (): FixedTimeTabState => {
     const { essence, timekeeper, dimension: { name } } = this.props;
     const shift = essence.timeShift.toJS();
-
     const timeFilter = essence.getEffectiveFilter(timekeeper).clauseForReference(name);
     if (timeFilter && timeFilter instanceof FixedTimeFilterClause && !timeFilter.values.isEmpty()) {
-      const { start, end } = timeFilter.values.get(0);
-      return { start, end, shift };
+      if (timeFilter.values.size === 1) {
+        const { start, end } = timeFilter.values.get(0);
+        return { start, end, shift, activeMonthStartDate: null, hoverTimeRange: null, selectionSet: false, dateRangePickers: List(), timeRanges: List(), numTimeRanges: 1 };
```

Comment:
It is complicated. You have `start` and `end` for one range, but a list of `start`s and `end`s for multiple values. Single list should suffice.